modules/logging-loki-zone-fail-recovery.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-loki-zone-fail-recovery_{context}"]
+= Recovering Loki pods from failed zones
+
+In {product-title} a zone failure happens when specific availability zone resources become inaccessible. Availability zones are isolated areas within a cloud provider's data center, aimed at enhancing redundancy and fault tolerance. If your {product-title} cluster isn't configured to handle this, a zone failure can lead to service or data loss.
+
+Loki pods are part of a link:https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/[StatefulSet], and they come with Persistent Volume Claims (PVCs) provisioned by a `StorageClass`. Each Loki pod and its PVCs reside in the same zone. When a zone failure occurs in a cluster, the StatefulSet controller will automatically attempt to recover the affected pods in the failed zone.
+
+[WARNING]
+====
+The following procedure will delete the PVCs in the failed zone, and all data contained therein.  To avoid complete data loss the replication factor field of the `LokiStack` CR should always be set to a value greater than 1 to ensure that Loki is replicating.
```

Comment:
```suggestion
The following procedure deletes the PVCs in the failed zone, and all data contained therein.  To avoid complete data loss, the replication factor field of the `LokiStack` CR should always be set to a value greater than 1 to ensure that Loki is replicating.
```

Reply:
| To avoid complete data loss the replication factor field of the `LokiStack` CR should always be set to a value greater than 1 to ensure that Loki is replicating.

Should this be in the logging-loki-zone-aware-rep.adoc module, where you are creating the CR?

Reply:
No, the value for that field is set automatically when deployment size is selected and thus would not be less than 2 unless it had been manually changed. The warning is present here as a sort of "are you sure" check before the deletion of a PVC.